components/Init/src/virtio_con.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2022, UNSW (ABN 57 195 873 179)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
```

Comment:
Please add a comment to make it obvious what this file is.

Reply:
> Seems like there are a lot of common code between [components/VM_Arm/src/modules/virtio_con.c](https://github.com/seL4/camkes-vm/pull/36/files#diff-3265d75a21c89137c4647bd28d5d75b4db0b59e7e87a986636a1130557d545ea) and [components/Init/src/virtio_con.c](https://github.com/seL4/camkes-vm/pull/36/files#diff-98b5c9eefaed33c6cd04830d9d0157ce0ccd3c16fb0ad6ee5c1fa4782316ddf2), is it really necessary?

I've just been following the set up for the virtio backends in the VMM so far. These files need a refactor (in addition with virtio net). Handling the RX and TX is similar between the x86/ARM VMMs, and the majority of the difference comes from setting up the virtual IRQs for the guest OS.

Reply:
Please add a comment to say what this file is for!

Reply:
Please a add a big comment at the start of this file to say what this file is all about.